frontend/src/scenes/plugins/source/PluginSourceTabs.tsx

```diff
@@ -2,24 +2,30 @@ import React from 'react'
 import { BuiltLogic, useActions, useValues } from 'kea'
 import { LemonButton } from 'lib/components/LemonButton'
 import type { pluginSourceLogicType } from './pluginSourceLogicType'
+import { IconPlus } from 'lib/components/icons'
 
 export function PluginSourceTabs({ logic }: { logic: BuiltLogic<pluginSourceLogicType> }): JSX.Element {
-    const { setCurrentFile } = useActions(logic)
+    const { setCurrentFile, addFilePrompt } = useActions(logic)
```

Comment:
1. I added a "remove" button. It's not the greatest 🍞 in the world, but does what it says. 

![2022-10-04 16 06 54](https://user-images.githubusercontent.com/53387/193841218-569d5069-7d0c-4ff3-a467-dca372dc2c83.gif)

2. Initially I wanted to just hide `web.ts` if not present, and show the other 2-3 files (`plugin.json`, `index.ts`, plus `frontend.ts` if the flag is enabled) at all times. I now changed it to only show the files that are actually present. I think this is cleaner, as I had the worry I might accidentally add a blank "index.ts" with something before. As for adding arbitrary files, I think we should support that sooner or later. However ok, limited it to only the files we support for now.

3. Fixed

Reply:
This is kind of a regression in terms of the experience:
![2022-10-05 13 50 27](https://user-images.githubusercontent.com/4550621/194053923-dd4cc3b6-d84f-4d98-818f-843e5ac9dcca.gif)

At the very least this should be a dropdown with available files instead of a freeform `prompt()`. The idea with arbitrary files is very far out at the moment, so optimizing this is very premature.
Then also this placement of the "Delete" button doesn't seem super intuitive. The button appearing and disappearing moves the "Add new file" button around a lot, and there's a difference in scope between "Add some new file" and "Delete this open file" that doesn't sit well with me for some reason…

I'd still argue for keeping this simpler and using the same mechanism as with `frontend.tsx` – users with the feature flag on will see `web.ts`, others won't. This should still scale well here.

Reply:
This is kind of a regression in terms of the experience:
![2022-10-05 13 50 27](https://user-images.githubusercontent.com/4550621/194053923-dd4cc3b6-d84f-4d98-818f-843e5ac9dcca.gif)

At the very least this should be a dropdown with available files instead of a freeform `prompt()`. The idea with arbitrary files is very far out at the moment, so optimizing this is very premature.
Then also this placement of the "Delete" button doesn't seem super intuitive. The button appearing and disappearing moves the "Add new file" button around a lot, and there's a difference in scope between "Add some new file" and "Delete this open file" that doesn't sit well with me for some reason…

I'd still argue for keeping this simpler and using the same mechanism as with `frontend.tsx` – users with the feature flag on will see `web.ts`, others won't. This should still scale well here.